core/native-bridge.ts

```diff
@@ -336,15 +336,15 @@ const initBridge = (w: any): void => {
     };
 
     const serializeConsoleMessage = (msg: any): string => {
-      if (typeof msg === 'object') {
-        try {
-          msg = JSON.stringify(msg);
-        } catch (e) {
-          // ignore
+      try {
+        if (typeof msg === 'object') {
+            msg = JSON.stringify(msg);
         }
+        return String(msg);
+      }
+      catch (e) {
+        return '??';
```

Comment:
This should probably return a blank string, if that will also avoid the error. Or perhaps an error message to the console and a blank string.

Reply:
yup, seems I left the ?? as was not sure what should be returned. I'm definitely ok with empty string unless there is any better suggestion.